cmd/bake/bake.go

```diff
@@ -19,11 +21,14 @@ type Options struct {
 	// Endpoint is the service endpoint to reach out to
 	Endpoint string
 
-	// URL is the git repo URL that will be sourced via 'pizza bake'
-	URL string
+	// URLs are the git repo URLs that will be source via 'pizza bake'
```

Comment:
nit: The grammar here is off

```suggestion
	// URLs are the git repo URLs that will be sourced via 'pizza bake'
```